### PR TITLE
Add errors.messages.in translation

### PR DESCRIPTION
**Pull request by jmschp**
Add translations:

- Portuguse `pt.errors.messages.in` to `"deve estar em %{count}"`
- Brazilian Portuguse `pt-br.errors.messages.in` to `"deve estar em %{count}"`
- Spanish `es.errors.messages.in` to `"debe estar en %{count}"`

### CODE REVIEW

File: rails/locale/es.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ es:
     - vie
     - sáb
     abbr_month_names:
-    - 
+    -
```
Comment (jmschp) [inline rails/locale/es.yml:20]: My VS Code automatically cleans trailing whitespaces on save. So it created this changes, and I only noticed after the PR was created.

I don't think we need the whitespaces, but let me know if you want me to add the back again.

**Comment by pama** [comment]
@jmschp Could you update the CHANGELOG, unreleased section? Thanks!

**Comment by jmschp** [comment]
> @jmschp Could you update the CHANGELOG, unreleased section? Thanks!

done!

**Review by sunny — APPROVED** [review]

**Comment by pama** [comment]
Thanks!

**Pull request merged**